src/reformat_libyuv.c

```diff
@@ -406,6 +407,25 @@ avifResult avifImageYUVToRGBLibYUV8bpc(const avifImage * image,
             }
             return AVIF_RESULT_OK;
         }
+    } else if (rgb->format == AVIF_RGB_FORMAT_RGB_565) {
+        // AVIF_RGB_FORMAT_BGR   *ToRGB565Matrix  matrixYUV
+
+        if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV420) {
```

Comment:
as of now only 420to565matrix is supported in libyuv.

it is trivial to add 422 and probably 444 as well. i will try to add those to libyuv first and then update the code here.